release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -0,0 +1,1677 @@
+|Supported ^[1]^
+|Supported ^[1]^,^[2]^
+
+|Persistent storage using Raw Block
+|Supported ^[1]^
+|Supported ^[1]^,^[2]^
+
+|Persistent storage using EDEV/FBA
+|Supported ^[1]^
+|Supported ^[1]^,^[2]^
+|====
+[.small]
+--
+1. Persistent shared storage must be provisioned by using either {rh-storage-first} or other supported storage protocols.
+2. Persistent non-shared storage must be provisioned by using local storage, such as iSCSI, FC, or by using LSO with DASD, FCP, or EDEV/FBA.
+--
+
+.Operators
+[cols="2,1,1",options="header"]
+|====
+|Feature |{ibm-power-name} |{ibm-z-name} and {ibm-linuxone-name}
+
+|{cert-manager-operator}
+|Supported
+|Supported
+
+|Cluster Logging Operator
+|Supported
+|Supported
+
+|Cluster Resource Override Operator
+|Supported
+|Supported
+
+|Compliance Operator
+|Supported
+|Supported
+
+|Cost Management Metrics Operator
+|Supported
+|Supported
+
+|File Integrity Operator
+|Supported
+|Supported
+
+|HyperShift Operator
+|Technology Preview
+|Technology Preview
+
+|{ibm-power-server-name} Block CSI Driver Operator
+|Supported
+|Unsupported
+
+|Ingress Node Firewall Operator
+|Supported
+|Supported
+
+|Local Storage Operator
+|Supported
+|Supported
+
+|MetalLB Operator
+|Supported
+|Supported
+
+|Network Observability Operator
+|Supported
+|Supported
+
+|NFD Operator
+|Supported
+|Supported
+
+|NMState Operator
+|Supported
+|Supported
+
+|OpenShift Elasticsearch Operator
+|Supported
+|Supported
+
+|Vertical Pod Autoscaler Operator
+|Supported
+|Supported
+|====
+
+.Multus CNI plugins
+[cols="2,1,1",options="header"]
+|====
+|Feature |{ibm-power-name} |{ibm-z-name} and {ibm-linuxone-name}
+
+|Bridge
+|Supported
+|Supported
+
+|Host-device
+|Supported
+|Supported
+
+|IPAM
+|Supported
+|Supported
+
+|IPVLAN
+|Supported
+|Supported
+|====
+
+.CSI Volumes
+[cols="2,1,1",options="header"]
+|====
+|Feature |{ibm-power-name} |{ibm-z-name} and {ibm-linuxone-name}
+
+|Cloning
+|Supported
+|Supported
+
+|Expansion
+|Supported
+|Supported
+
+|Snapshot
+|Supported
+|Supported
+|====
+
+[id="ocp-release-notes-insights-operator-enhancements_{context}"]
+=== Insights Operator
+
+[id="ocp-release-notes-installation-and-update_{context}"]
+=== Installation and update
+
+[id="ocp-release-notes-olm_{context}"]
+=== Operator lifecycle
+
+[id="ocp-release-notes-osdk_{context}"]
+=== Operator development
+
+[id="ocp-release-notes-openshift-cli_{context}"]
+=== OpenShift CLI (oc)
+
+[id="ocp-release-notes-oci"]
+=== {oci-first}
+
+[id="ocp-release-notes-machine-config-operator_{context}"]
+=== Machine Config Operator
+
+[id="ocp-release-notes-machine-management_{context}"]
+=== Machine management
+
+[id="ocp-release-notes-monitoring_{context}"]
+=== Monitoring
+
+[id="ocp-release-notes-networking_{context}"]
+=== Networking
+
+[id="ocp-release-notes-nodes_{context}"]
+=== Nodes
+
+[id="ocp-release-notes-postinstallation-configuration_{context}"]
+=== Postinstallation configuration
+
+[id="ocp-release-notes-registry"]
```

Comment:
This doesn't have to be done in this PR if it'll get in the way too much, but please comb through this assembly at some point and add contexts to any IDs that are missing them:
```suggestion
[id="ocp-release-notes-registry_{context}"]
```

Reply:
Thanks. I updated three instances!